styleguide/_themes/derek/scss/_main.scss

```diff
@@ -13,3 +13,15 @@
     margin-top: 89px;
   }
 }
+
+.feature-container {
+  h1, h2, h3 { font-family: $subhead; font-weight: 300; text-transform: none; }
```

Comment:
noooo sir!
no redefining heading tags. no exceptions. Just switch them in the markup to h4, h5 and h6